cmd/maestro/server/grpc_server.go

```diff
@@ -123,6 +129,45 @@ func (svr *GRPCServer) Publish(ctx context.Context, pubReq *pbv1.PublishRequest)
 	return &emptypb.Empty{}, nil
 }
 
+func (svr *GRPCServer) Subscribe(subReq *pbv1.SubscriptionRequest, subServer pbv1.CloudEventService_SubscribeServer) error {
+	topicSplits := strings.Split(subReq.Topic, "/")
+	if len(topicSplits) != 5 {
+		return fmt.Errorf("invalid subscription topic %s", subReq.Topic)
+	}
+
+	source, clusterName, statusSub := topicSplits[1], topicSplits[3], topicSplits[4]
+	if source == "" || clusterName == "" || statusSub != "status" {
+		// TODO: validate source and clusterName
+		return fmt.Errorf("invalid subscription topic %s", subReq.Topic)
+	}
+
+	eventClient := event.NewEventClient(clusterName)
+	svr.eventHub.Register(eventClient)
```

Comment:
how about subscriber := svr.eventHub.Register(clusterName)? and  svr.eventHub.Unregister(clusterName)?
eventClient is quite confusing because it is only used to subrcibe.